hexa/pipelines/models.py

```diff
@@ -272,6 +307,7 @@ class Meta:
     outputs = models.JSONField(null=True, blank=True, default=list)
     run_logs = models.TextField(null=True, blank=True)
     current_progress = models.PositiveSmallIntegerField(default=0)
+    send_mail_notifications = models.BooleanField(null=True, default=False)
```

Comment:
@pvanliefland shouldn't it be null=False and been applied to all existing runs in the migration as False ?

Reply:
Sorry @qgerome @cheikhgwane just saw this now. I'd remove `null=True` altogether (`null` is `False` by default). 

The `default=False` param should be enough for the migrations.

Reply:
Ping @cheikhgwane